### PR TITLE
editing developer sign and adding more

### PR DESCRIPTION
**Pull request by souperneon**
- edited developer sign up cta, feature boxes
- trimmed css styles for sign up cta
- added author credit boxes

### CODE REVIEW

File: src/stylesheets/common/_cta.scss [hunk 1]
```diff
@@ -17,9 +17,7 @@
 .cta-text {
   // Match text size and line height to button.
```
Comment (louh) [inline src/stylesheets/common/_cta.scss]: if we don't need font-size and line-height here anymore please remove this comment

File: src/site/ui-components/signup-cta/index.html [hunk 2]
```diff
@@ -1,22 +1,17 @@
 <div id="signup-cta" class="headroom-large">
-  <h5>Developer Sign up/ Sign in Call to Action</h5>
+  <h5>Developer Sign up Call-to-Action</h5>
   <p>
-    The sign up CTA is used in different places on the site to prompt signing up for a Mapzen developer account or a key to a particular project. On project pages, it is customized to use the current project name which makes it more targeted.
+    The sign up CTA is used in different places on the site to prompt signing up for a Mapzen developer account or a key to a particular project. There are two ways to use it - on project specific pages substitute "Mapzen services" with 'Project name' like "Mapzen Search" or just use the more general version like below.
   </p>
   <div data-xrayhtml>
-    <div class="cta-container headroom">
+    <div class="row cta-container">
```
Comment (louh) [inline src/site/ui-components/signup-cta/index.html]: is the .row class necessary? what does it do here that it didn't do before?

File: src/stylesheets/common/_cta.scss (hunk 1 above)
Comment (souperneon) [inline src/stylesheets/common/_cta.scss]: will do.

File: src/site/ui-components/signup-cta/index.html (hunk 2 above)
Comment (souperneon) [inline src/site/ui-components/signup-cta/index.html]: @hanbyul-here I know we talked about always having the .row class?  Chances are I have used it incorrectly

Comment (souperneon) [inline src/site/ui-components/signup-cta/index.html]: ok. so we don't need this because we are doing the button placement using table-cell. I'll fix this

**Pull request merged**